### ISSUE TITLE
Stuck after Detected disconnect (Out of Sync During opponent Search)

### ISSUE THREAD

**Issue by subgrenadier** [norecha/pokubot#44]
During opponent search, (click next next next) the game shall out of sync and required to be restart. Pokubot able to identify but after the game start again, it does nothing. Please help. Thank u

**Comment by norecha** [comment]
put the log file please. you may need to wait 15 sec after disconnect for it to detect

**Comment by roinoir** [comment]
[03.05.15 11:01:38 PM] aok.coc.launcher.DisconnectChecker run INFO: Detected disconnect. 
[03.05.15 11:01:38 PM] aok.coc.util.RobotUtils leftClickWin32 FINEST: clicking 436 400 
[03.05.15 11:06:51 PM] aok.coc.launcher.DisconnectChecker run INFO: Detected disconnect. 
[03.05.15 11:06:51 PM] aok.coc.util.RobotUtils leftClickWin32 FINEST: clicking 434 401 
[03.05.15 11:08:47 PM] aok.coc.controller.MainWindowController$4 handle WARNING: runner is cancelled. 
[03.05.15 11:08:47 PM] aok.coc.launcher.BotLauncher loop INFO: Interrupted by user. 
[03.05.15 11:08:47 PM] aok.coc.launcher.DisconnectChecker run SEVERE: dc checker: sleep interrupted 
java.lang.InterruptedException: sleep interrupted
	at java.lang.Thread.sleep(Native Method)
	at aok.coc.launcher.DisconnectChecker.run(DisconnectChecker.java:68)
	at java.lang.Thread.run(Unknown Source)

got 3 hours straight for the disconect detected

**Comment by subgrenadier** [comment]
The issue had solved. PokuBot reload itself and works fine. Thanks

**Comment by roinoir** [comment]
mine didn't reload so il wait since our problem come from the same place

**Comment by subgrenadier** [comment]
I monitor the bot and find out that sometime it can reload, sometime it just detect out of sync error and the game restart and the bot stop. Please see the status when reloading in the url : http://i59.tinypic.com/16j1zmg.jpg

**Comment by roinoir** [comment]
send the log fil sub
that file is located in your poku folder (the bot one)
the log is important because it show the exact error code and action of the bot who help nercha to find the error

**Comment by norecha** [comment]
There is a bug that may happen rarely. If bot dies(barracks configuration is wrong for example) and you click stop, it doesn't reset some variables and next time you get disconnected, it won't reload.
If this happens just close the bot and run it again after setup is done.
I assume it can also happen when you get disconnected due to sync error let's say, and you just click stop manually before letting bot detect it.
To sum up, don't use stop button that often, let bot detect it

**Comment by roinoir** [comment]
seems the error happen when the bot got disconected while looking for base 
the bot is stuck in attack/stade and don't return normaly in stade/train or stade/idle
